interview-questions.md

```diff
@@ -8,42 +8,88 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: Object-oriented programming is a development structure that forms code and functionality around using objects. This can be seen evidently in the lanuguage Ruby where everthing is essentialy an object. 
 
-Researched answer:
+Researched answer: a softeware development model that structures design around objects as opposed to functions and logic. An object is data that has unique behavior and its own attributes. 
 
-1. What is the difference between a Float and an Integer in Ruby?
+answers via: https://www.techtarget.com/searchapparchitecture/definition/object-oriented-programming-OOP#:~:text=Object%2Doriented%20programming%20(OOP)%20is%20a%20computer%20programming%20model,has%20unique%20attributes%20and%20behavior.
 
-Your answer:
 
-Researched answer:
+2. What is the difference between a Float and an Integer in Ruby?
 
-1. Ruby has an implicit return. What does that mean?
+Your answer: In Ruby there are two ways on desplaying a number. Float is the keyword that allows users to utilize decimal points. Integar is the keyword that refers to whole numbers. It will not give decimals and is the same as the number keyword, but integer is for the language Ruby. 
```

Comment:
While you definitely indicated differences, one important point is that Integer and Float are unique classes in Ruby. So a 1 is an instance of class Integer and 3.14 is an instance of class Float.